aion/web/elm/src/Panel/View.elm

```diff
@@ -15,22 +16,56 @@ panelView model =
     div []
         [ h3 [] [ text "Create new question for certain category:" ]
         , form []
-            [ p [] [ text "Enter question content below:" ]
-            , input [ placeholder "How much is 2+2?", onInput SetNewQuestionContent ] []
-            , p [] [ text "Enter answer below, separate all posibilities with comma:" ]
-            , input [ placeholder "4", onInput SetNewAnswerContent ] []
-            , p [] [ text "Select the category to which to add the question:" ]
-            , Select.from (listRooms model.rooms) SetNewAnswerCategory
+            [ questionFormElement model.panelData.createQuestionForm
+            , answersFormElement model.panelData.createQuestionForm
+            , subjectFormElement model.panelData.createQuestionForm (listRooms model.rooms)
             , input [ type_ "button", value "submit", onClick CreateNewQuestionWithAnswers ] []
             ]
         ]
 
 
+questionFormElement : Forms.Form -> Html Msg
+questionFormElement form =
+    div []
+        [ p [] [ text "Enter question content below:" ]
+        , input
+            [ placeholder "How much is 2+2?"
+            , onInput (UpdateCreateQuestionForm "question")
+            , value (Forms.formValue form "question")
+            ]
+            []
+        , small [] [ text (Forms.errorString form "question") ]
+        ]
+
+
+answersFormElement : Forms.Form -> Html Msg
+answersFormElement form =
+    div []
+        [ p [] [ text "Enter answer below, separate all posibilities with comma:" ]
+        , input
+            [ placeholder "4,cztery"
+            , onInput (UpdateCreateQuestionForm "answers")
+            , value (Forms.formValue form "answers")
+            ]
+            []
+        , small [] [ text (Forms.errorString form "answers") ]
+        ]
+
+
+subjectFormElement : Forms.Form -> List String -> Html Msg
+subjectFormElement form roomList =
+    div []
+        [ p [] [ text "Select the category to which to add the question:" ]
+        , Select.from roomList (UpdateCreateQuestionForm "subject")
+        , small [] [ text (Forms.errorString form "subject") ]
+        ]
+
+
 listRooms : WebData RoomsData -> List String
 listRooms result =
     case result of
         RemoteData.Success roomsData ->
-            List.map (\room -> room.name) roomsData.data
+            "" :: List.map (\room -> room.name) roomsData.data
```

Comment:
Validation library does not provide default values. So this is a dirty hack to have empty string at the beginning to force user to choose something. If the user would like to create a new question to the first displayed category he would need to choose something else and get back to it and then hit submit.